Assets/Scripts/Utils/NetworkSimulatorUIMediator.cs

```diff
@@ -91,11 +95,41 @@ void OnPresetChanged(int optionIndex)
         {
             m_NetworkSimulator.ChangeConnectionPreset(m_SimulatorPresets[m_PresetsDropdown.options[optionIndex].text]);
         }
+
+        void OnScenarioChanged(int optionIndex)
+        {
+            var scenarioName = m_ScenariosDropdown.options[optionIndex].text;
+            NetworkScenario scenario = null;
+            switch (scenarioName)
+            {
+                case k_None:
+                    break;
```

Comment:
yes, it calls its Dispose() method automatically. The rest is handled by garbage collection, but since we don't change scenarios frequently, the performance costs should be negligible here